src/Symfony/Component/Form/FormConfigBuilder.php

```diff
@@ -69,18 +66,19 @@ class FormConfigBuilder implements FormConfigBuilderInterface
      * @throws InvalidArgumentException if the data class is not a valid class or if
      *                                  the name contains invalid characters
      */
-    public function __construct(?string $name, ?string $dataClass, EventDispatcherInterface $dispatcher, array $options = [])
-    {
+    public function __construct(
+        ?string $name,
+        private ?string $dataClass,
```

Comment:
I would revert this change. In other places where we check the passed value to (for example) throw exceptions we don't use CPP.